### ISSUE TITLE
Final Peer Review - cyz9

### ISSUE THREAD

**Issue by zhouc1234** [SZ559/Analysis-of-Learning-Rate-for-Deep-Neural-Networks#11]
For the analysis of learning rate for deep neural networks algorithm development project, the team aims to optimize gradient descent methods by imposing their “RADAMS” algorithm. The team also conducts in-depth literature reviews on the gradient descent to help develop their algorithm
Pros:
- The team does well to introduce their topic and explain the background as well as the purpose of their project
- I really appreciated the explanation and use of the literature review detailing each of the algorithms (ex. Momentum based, adaptive learning)
- I also liked the development of the RADAMS algorithm and the step by step process

Room for Improvement:
- more explanation and equations could’ve been used to detail the original SGD in the beginning
- A more thorough explanation of each of the datasets and features used would be better
